terraform/aks/application.tf

```diff
@@ -31,8 +31,88 @@ module "application_configuration" {
   }
 }
 
+# Run database migrations
+# https://guides.rubyonrails.org/active_record_migrations.html#preparing-the-database
+# https://github.com/ilyakatz/data-migrate
+# Terraform waits for this to complete before starting web_application and worker_application
```

Comment:
### TL;DR
No need to include a link to the [data_migrate](https://github.com/ilyakatz/data-migrate) gem here – this app doesn't use it, so it's not relevant.

```suggestion
# Run database migrations
# https://guides.rubyonrails.org/active_record_migrations.html#preparing-the-database
# Terraform waits for this to complete before starting web_application and worker_application
```

### The long story

At the time of https://github.com/DFE-Digital/itt-mentor-services/pull/893, we were using data_migrate to differentiate between 'schema' migrations and 'data' migrations. Therefore it seemed relevant to include a comment in this Terraform config to explain why we were running two Rake commands (`db:prepare` for Rails, and `data:migrate` for data_migrate).

However we later decided it was causing more pain that it was worth, so we dropped it in https://github.com/DFE-Digital/itt-mentor-services/pull/1022.

(Although now I look at it, we've still got that same comment hanging around in `main`. We forgot to tidy it up. So I'll remove it now!)